modules/builds-using-build-volumes.adoc

```diff
@@ -68,7 +68,7 @@ spec:
 
 [NOTE]
 ====
-The shared resource CSI driver is supported as **Tech Preview** only.
+The Shared Resource CSI Driver is supported as **Tech Preview** only.
```

Comment:
```suggestion
The Shared Resource CSI Driver is supported as **Technology Preview** only.
```
I think that "Tech Preview" be changed to "Technology Preview" throughout this PR. I did a search in the OCP docs and found 4-5 instances of "Tech Preview" in the published docs, but the SSG definitely spells out "Technology Preview" in its boilerplate template. Not 100% sure on what's acceptable, but shortening Technology to Tech doesn't feel right.

Reply:
Makes sense to me! Updated. Thanks!